server/hedley/modules/custom/hedley_municipality/hedley_municipality.module

```diff
@@ -17,6 +17,28 @@ function hedley_municipality_ctools_plugin_directory($module, $plugin) {
 }
 
 /**
+ * Implements hook_init().
+ */
+function hedley_municipality_init() {
+  if (drupal_is_front_page() && !drupal_is_cli()) {
+    // Redirect from homepage to the group node view when on homepage.
+    hedley_municipality_redirect_to_group_view();
+  }
+}
+
+/**
+ * Redirect to the group node view when inside `og_context`.
+ */
+function hedley_municipality_redirect_to_group_view() {
```

Comment:
I think you can actually remove this function, and move all logic to `hedley_municipality_init`. MWG